library/core/functions.general.php

```diff
@@ -3110,6 +3110,32 @@ function safeImage($ImageUrl, $MinHeight = 0, $MinWidth = 0) {
     }
 }
 
```

Comment:
https://github.com/vanilla/vanilla/blob/fa1c31f54f469ca53eaf1d59596f7a4585e780c9/library/core/functions.general.php#L3705

The function already exists!